docs/notebooks/ask_tell_optimization.pct.py

```diff
@@ -0,0 +1,166 @@
+# %% [markdown]
+# # Ask-Tell Optimization Interface
+
+# %% [markdown]
+# In this notebook we will illustrate the use of Ask-Tell interface in Trieste. It might be useful in the cases when you want to have greater control of the optimization loop, or if letting Trieste manage this loop is impossible.
```

Comment:
In this notebook we will illustrate the use of **an** Ask-Tell interface in Trieste. **It is** useful for cases where you want to have greater control of the optimization loop, or **when letting** Trieste manage this loop is impossible.